schemas/EGA.experiment.json

```diff
@@ -2,12 +2,324 @@
+            "examples": [ "MC-4C", "UMI-4C", "single nucleus RNA sequencing", "RNA-seq of coding RNA from single cells", "Quant-seq", "DNA-seq", "ATAC-seq", "DNase Hi-C", "scDNase-seq", "in situ HiC", "exome sequencing" ]
+          }
+        },
+        "anyOf": [
+          {
+            "title": "Assay subtypes match DNA/RNA assays",
+            "properties": {
+              "assay_subtype": {
+                "anyOf": [
+                  {
+                    "title": "DNA-Assay subtype controlled vocabulary (CV) list",
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_dna.json"
+                  },
+                  {
+                    "title": "RNA-Assay subtype controlled vocabulary (CV) list",
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_rna.json"
+                  }
+                ]                 
+              }
+            }
+          },
+          {
+            "title": "Assay subtypes match array/sequencer assays",
+            "oneOf": [
+              {
+                "title": "Assay type and subtype terms are from the array CV list",
+                "properties": {
+                  "assay_type": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_type_by_array.json" 
+                  },
+                  "assay_subtype": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_array.json" 
+                  }
+                }
+              },
+              {
+                "title": "Assay type and subtype terms are from the sequencer CV list",
+                "properties": {
+                  "assay_type": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_type_by_sequencer.json" 
+                  },
+                  "assay_subtype": {
+                    "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/controlled_vocabulary_schemas/EGA.cv.assay_subtype_by_sequencer.json" 
+                  }
+                }
+              }
+            ]
+          }
+        ]
+      },
+
+      "assayed_molecule": {
+        "type": "string",
+        "title": "Assayed molecule of the experiment [EFO:0002772]",
+        "description": "The human readable ID (e.g. DNA assay), chosen from a list of controlled vocabulary (CV), of the assayed molecule. If you cannot find your term in the CV list, please create an issue at our [metadata GitHub repository](https://github.com/EbiEga/ega-metadata-schema) proposing its addition.",
+        "enum": ["DNA assay", "RNA assay", "Metabolomic profiling", "Protein assay"],
```

Comment:
These enum values are types of assays rather than types of molecules. I suggest using the list of molecules listed under the [EFO:0004446](http://www.ebi.ac.uk/efo/EFO_0004446) ontology.

Reply:
Although I agree that being more specific could be beneficial, under EFO:0004446 ontology there are no references to synthetic DNA, proteins, metabolites...

Would it be better to just add DNA and RNA, and leave other categories out? Another thing we could do is something similar to the *_type and *_subtype classifications we used in other places: having a general molecule identifier (DNA, RNA, metabolite and protein) and a subtype (e.g. messenger RNA).

Reply:
I suggest having just a type (also having subtype is too complex right now) listing all the molecules under [EFO:0004446](http://www.ebi.ac.uk/efo/EFO_0004446) (maybe ignore the "adhesion factor") and also add two terms for protein and metabolite (although we really shouldn't get this data)